README.md

```diff
@@ -41,21 +46,22 @@ One of the goals of this library is to make sure that the it is easy to use.  Ev
 **Velaptor** has the ability to load texture atlas image and JSON data for texture atlas type rendering as well as single image content and sound content.  The content loading system is flexible and has an API that has the ability to be extended to load custom content for your media applications and games.
 
 ### Content Caching
-Loaded content such as images and sounds are cached for reuse and performance reasons.  If the same content is loaded from disk after it has already been loaded, the same image content will be used for rendering.  This is the same for sounds and texture atlas data.  Custom content loading will not come with caching and will have to be added.
+Loaded content such as images and sounds are cached for performance reasons.  If the same content attempting to be loaded from disk after it has already been loaded, that same content will be used from memory instead of being reloaded.  This is the same for sounds and texture atlas data.  Custom content loading will not come with caching and will have to be added manually.
```

Comment:
reword
```suggestion
Loaded content such as images and sounds are cached for performance reasons.  If the same content is attempting to be loaded from a disk after it has already been loaded, that same content will be used from memory instead of being reloaded.  Sounds and texture atlas data have the same functionality.  Custom content loading will not come with caching and will have to be added manually.
```

Reply:
You missed the "a" in front of "disk"